mkdocs.yml

```diff
@@ -1,21 +1,10 @@
 ---
-site_name: 'Rocky Linux Documentation'
+site_name: 'Documentation'
 site_url: 'https://docs.rockylinux.org'
 docs_dir: 'docs/docs'
 repo_url: https://github.com/rocky-linux/documentation
 repo_name: rocky-linux/documentation
-edit_uri: 'edit/main/docs/'
```

Comment:
@wsoyinka -- I know having an edit link was something you specifically wanted, so is this a thing you're happy removing?

Reply:
@hbjydev   -  Thanks !!   Edit links are important.  We don't want this removed !!!!!!

Reply:
@justasojourner can you re-introduce this line again? Thanks.

Reply:
I _think_ I spoke about this with Steven. This is actually directly editing content ‘in line’ that is **directly editing the GitHub source***. Perhaps you were not aware of that? 

The workflow process for making changes to content is to do on a forked copy of repo on an author’s machine and then submit a PR, even those people who have authority on the repo, e.g. Steven @sspencerwire don’t edit directly. 

Go to any page on the production website (**while you are logged out of GitHub**) and click on the pencil edit icon and see what comes up. That error is what everyone sees - and given that even those who could, don’t, why have the edit button there to confuse everyone?

Reply:
@justasojourner   - Thanks for clarifying this point.   All this while I had been browsing the docs.r.o site, I always had an authenticated github session going and so I never noticed the undesirable behaviour you just pointed out.   Your careful attention is much appreciated :-).

Reply:
@wsoyinka no problemo - so you will advise @hbjydev that this option is not needed/desired so my change removing the edit URI (which will remove the pencil edit icon) will stand.

What about the other topic re: localsearch: 
* which is not needed
* removing of which has no impact on searching
* is not being loaded anyway

You will advise Hayden that that change stands also?

In which case there is no road-blocks for the PR.